### DISCUSSION TITLE
質問への編集リクエスト機能（もしくはコメント）

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#533] [要望・アイデア・不具合]
現在、記事には編集リクエストを送る事が可能ですが、質問にはそれができません。
質問やMarkdownに不慣れな人だと、
・タイトルで質問の概要が分かり辛い
・マークダウンの記述がおかしく、ソースの表示が崩れている
といった事がしばしば見受けられ、これを回答で指摘するのもなんか違うかな、と感じます。（回答は回答内容以外の事を書きたくない）
質問への編集リクエストやコメントができれば、こう直すといいのではという提案ができるので、将来的に可能であれば実現していただけると利便性が向上すると思います。

**Comment by tanaka-asami415** [comment]
@radian-jp 
ご意見ありがとうございます！

> これを回答で指摘するのもなんか違うかな、と感じます。（回答は回答内容以外の事を書きたくない）

ご状況とても共感できるものの、編集リクエスト機能は実装コストが大きく優先度については検討中となります。
将来的に編集リクエスト機能、もしくは他の方法で代替機能がご用意できないか検討させていただきます🙇
